app/components/MazemapEmbed/MazemapAccordion.tsx

```diff
@@ -36,22 +27,24 @@ export const MazemapAccordion = ({ defaultOpen = false, ...props }: Props) => {
                 Kart
               </Flex>
               <Icon
-                onPress={() => {
-                  onClick();
-                  setMapIsOpen(!mapIsOpen);
-                }}
+                onPress={onClick}
                 iconNode={<ChevronRight />}
                 className={rotateClassName}
               />
             </div>
           )}
         >
-          <MazemapEmbed {...props} className={styles.mazemapEmbed} />
+          <Flex column>
+            <MazemapEmbed {...props} className={styles.mazemapEmbed} />
+            <div className={styles.mazemapLinkContainer}>
```

Comment:
```suggestion
            <Flex className={styles.mazemapLinkContainer}>
```
Could save some lines of css